components/Player/NotesTable.tsx

```diff
@@ -1,24 +1,144 @@
+    const response = await fetch(`/api/notes/delete?id=${note.id}`, {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    if (!response.ok) {
+      throw await response.json();
+    }
+    refreshProfile();
+    toasty();
+  }
+  return (
+    <>
+      <Menu>
+        {({ open }) => (
+          <>
+            <Menu.Button className="relative focus:outline-none flex align-center justify-center">
+              <Icon type="more" className="h-5 ml-4 fill-current" />
+            </Menu.Button>
+            <Transition
+              show={open}
+              enter="transition duration-100 ease-out"
+              enterFrom="transform scale-95 opacity-0"
+              enterTo="transform scale-100 opacity-100"
+              leave="transition duration-75 ease-out"
+              leaveFrom="transform scale-100 opacity-100"
+              leaveTo="transform scale-95 opacity-0"
+            >
+              <Menu.Items
+                className="absolute z-10 border-medium-gray shadow-lg bg-white rounded-md pt-12 focus:outline-none flex flex-col text-unselected font-semibold text-sm w-32"
+                style={{ borderWidth: 1, transform: "translateY(-32px)" }}
+                static
+              >
+                <Icon type="more" className="h-5 ml-4 -mt-10 fill-current" />
+                <Menu.Item>
+                  {({ active }) => (
+                    <button
+                      type="button"
+                      className={`flex items-center w-full px-4 py-2 font-medium ${
+                        active ? "bg-button text-dark" : ""
+                      }`}
+                      onClick={editNote}
+                    >
+                      <Icon type="edit" className="h-4 mr-3 fill-current" />
+                      <p className="justify-self-start">Edit</p>
+                    </button>
+                  )}
+                </Menu.Item>
+                <Menu.Item>
+                  {({ active }) => (
+                    <button
+                      type="button"
+                      className={`flex items-center w-full px-4 py-2 rounded-b-md font-medium ${
+                        active ? "bg-button text-dark" : ""
+                      }`}
+                      onClick={deleteNote}
+                    >
+                      <Icon type="delete" className="h-4 mr-3 stroke-current" />
+                      <p className="justify-self-start">Delete</p>
+                    </button>
+                  )}
+                </Menu.Item>
+              </Menu.Items>
+            </Transition>
+          </>
+        )}
+      </Menu>
+      <AddNote
+        addOrEdit="Edit"
+        authorId={currNote.authorId}
+        note={currNote}
+        modalOpen={edit}
+        closeModal={() => setEdit(false)}
+        toastMessage="Note updated!"
+        refresh={() => refreshProfile()}
+      />
+    </>
+  );
+};
+
```

Comment:
Let's move this into a separate file in the components folder so that it's a bit cleaner. Also let's fix up styling for the display of the Edit Button it looks out of place on my end 
<img width="1153" alt="Screen Shot 2021-04-25 at 3 33 09 AM" src="https://user-images.githubusercontent.com/57571626/115990237-f6c8a280-a576-11eb-9c7a-1fee65144b43.png">